week08/movie/src/components/Header.js

```diff
@@ -1,11 +1,14 @@
 import React from "react";
-import styled from "styled-components";
+import styled, { ThemeProvider } from "styled-components";
+import { useSelector } from "react-redux";
 
 const HeaderContainer = styled.div`
   display: flex;
   justify-content: space-between;
   padding: 0px 5px;
-  background-color: #222222;
+  background-color: ${(props) =>
+    props.theme.mode === "dark" ? "#ffffff" : "#222222"};
+
```

Comment:
Theme.js에서 darkmode와 lightmode에 사용할 header 색상을 정의해주었으므로, 
각 컴포넌트에서 색상을 가져다가 쓸 때는 조건문을 사용할 필요 없이, 
이미 정의해둔 객체 프로퍼티를 가져다가 쓰면 됩니다.

```suggestion
 background-color: ${(props) => props.theme.header};
```